src/idl_gen_grpc.cpp

```diff
@@ -60,12 +61,30 @@ class FlatBufMethod : public grpc_generator::Method {
   std::string name() const { return method_->name; }
 
   std::string GRPCType(const StructDef &sd) const {
-    return "flatbuffers::grpc::Message<" + sd.name + ">";
+    return "flatbuffers::grpc::Message<" + WrapInNameSpace(sd) + ">";
   }
 
-  std::string get_input_type_name() const { return (*method_->request).name; }
+  std::string WrapInNameSpace(const StructDef &sd) const {
```

Comment:
Can you make `FlatBufMethod` store a `BaseGenerator &` such that some of namespace functions in there can be reused here?

Reply:
OK I'll try.